src/modules/project/pages/projectFunding/utils/createAndSignAonClaimTransaction.ts

```
@@ -0,0 +1,115 @@
+  signedTxHex: Hex
+  gasCostWei: bigint
+}
+
+/**
+ * Encodes claim(processingFee) calldata.
+ * Prefer server-provided claimCalldata from aonClaimPrepare when available.
+ */
+export const encodeAonClaimCalldata = (params: {
+  claimCalldata?: string | null
+  processingFeeSats?: number
+}): Hex => {
```

ESLint

[error] Replace `⏎··claimCalldata?:·string·|·null⏎··processingFeeSats?:·number⏎` with `·claimCalldata?:·string·|·null;·processingFeeSats?:·number·`

(prettier/prettier)